Update README with Major Sparsify Announcement (#174)
neuralmagic/sparsify: README.md

    </a>
</p>

> ## 🚨 MAJOR ANNOUNCEMENT 🚨

Reviewer: ```suggestion
## 🚨 FEBRUARY 2023 MAJOR ANNOUNCEMENT 🚨
```

> We've been working on a major overhaul for Sparsify to help you build sparse models to target optimal general performance at scale with even more simplicity.

Reviewer: ```suggestion
Sparsify is getting a major overhaul with more features and simplicity to help you build sparse models to target optimal general performance at scale.
```

> Leverage powerful APIs through the Sparsify Platform that can help you optimize models from scratch or sparse transfer learn onto your own data, to target best-in-class inference performance on your deployment hardware.

Reviewer: ```suggestion
```

> We'll be able to share more in the coming weeks, but sign up for our [Early Access Waitlist](https://docs.neuralmagic.com/main/source/getstarted.html#sparsification) to be the first to try out our new sparsification offering.

Reviewer: ```suggestion
We will share more in the coming weeks. In the meantime, sign up for our [Early Access Waitlist](https://docs.neuralmagic.com/main/source/getstarted.html#sparsification) to be the first to try out our new sparsification offering.
```
